test/testdata/packager/import_subpackage/a/__package.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
```

Comment:
Another packager test -- which has this issue. Figure I'd leave it here for extra coverage; eventually we can isolate this error.